project/Game.py

```diff
@@ -71,9 +74,22 @@ def test_completed_game(self):
         self.game.end()
         self.assertTrue(self.game.ended, "Game Has Ended")
 
+class TestReOrderLandmarks(unittest.TestCase):
+    def setUp(self):
+        self.game = Game()
+
+    def test_change_order(self):
+        self.game.landmarks = ["Chicago", "Vegas", "Dallas"]
+        self.assertTrue(self.game.landmarks, "List is empty")
+        self.game.reorder_landmarks("Dallas", 1)
+        self.assertEqual("Dallas", self.game.landmarks[1], "Reorder did not move landmark to correct location")
```

Comment:
This only tests that the landmark is in the new location, it doesn't test that it was removed from the original location (possible duplication)